Goobi/src/org/kitodo/production/lugh/ld/AccessibleObject.java

```diff
@@ -11,6 +11,7 @@
 
 package org.kitodo.production.lugh.ld;
 
+import java.nio.BufferOverflowException;
```

Comment:
`BufferOverflowException` is used inside a [JavaDoc comment](https://github.com/matthias-ronge/kitodo-production/blob/483fefbf78c872002dedcf2cf8996731df13ce3e/Goobi/src/org/kitodo/production/lugh/ld/AccessibleObject.java#L30) and a good IDE mentioned the programmer that this symbol `BufferOverflowException` could not be resolved. Matthias try to fix that and as this it is related to pull request title.